src/pages/Pokemon/index.js

```diff
@@ -26,6 +27,19 @@ const Pokemon = () => {
     const catSpec = await cat.json();
     item.category = catSpec?.genera[7]?.genus;
 
+    const typeInfo1 = await fetch(item.types[0].type.url);
+    const typeStuff1 = await typeInfo1.json();
+    item.damageStuff1 = typeStuff1.damage_relations;
+    if (item.types.length > 0) {
+      try {
+        const typeInfo2 = await fetch(item.types[1].type.url);
```

Comment:
does the `types` array only ever have two items? If it sometimes has more then we could maybe use a loop to get them all

Reply:
Nah mate, no Pokemon is ever more than two types of pokemon.